packages/app/src/app/pages/Dashboard/Content/routes/Recent/RecentHeader.tsx

```diff
@@ -2,25 +2,24 @@ import track from '@codesandbox/common/lib/utils/analytics';
 import { CreateCard, Stack, Text } from '@codesandbox/components';
 import { useWorkspaceAuthorization } from 'app/hooks/useWorkspaceAuthorization';
 import { useWorkspaceSubscription } from 'app/hooks/useWorkspaceSubscription';
-import { useActions, useAppState } from 'app/overmind';
+import { useActions } from 'app/overmind';
 import { EmptyPage } from 'app/pages/Dashboard/Components/EmptyPage';
 import { UpgradeBanner } from 'app/pages/Dashboard/Components/UpgradeBanner';
 import React from 'react';
 
 export const RecentHeader: React.FC<{ title: string }> = ({ title }) => {
-  const { activeTeam } = useAppState();
   const actions = useActions();
   const { isFree } = useWorkspaceSubscription();
   const {
     isTeamSpace,
     isPersonalSpace,
     isTeamViewer,
   } = useWorkspaceAuthorization();
-  const showUpgradeBanner = isFree && isTeamSpace;
+  const showUpgradeBanner = isPersonalSpace || (isFree && isTeamSpace);
```

Comment:
Do we want to show the banner when all teams have subscriptions or trials?

Reply:
@tristandubbeld that's a great question. We shouldn't... Thank you for raising it 🙇🏽‍♀️

Reply:
[5e826f6](https://github.com/codesandbox/codesandbox-client/pull/7491/commits/5e826f69719117b62a03e1cd57bd8dce9ca97496)